packages/react-ui/src/theme.js

```diff
@@ -28,6 +44,14 @@ export const colors = {
 colors.primaryText = colors.white;
 colors.secondaryText = colors.white;
 
+export const highContrastColors = Object.entries(colors).reduce(
```

Comment:
We could do the same using a level approach were use can choose is level 3 / 7 / 12

Reply:
In fact I'm going to rename this one BlackAndWhiteTheme and next month we'll add a real highContrast mode. We already are in 3 so we only miss a 7 I guess but another time

Reply:
Oh sorry, the comment was not meant to be actionnable now but more a thought for later